js/weapon_requirements.js

```diff
@@ -1,379 +1,493 @@
 WEAPONS = {
-    "Battle Axe": {"str": 12, "dex": 8, "infusible": true},
```

Comment:
Can you please keep the original formatting to make the diff smaller, i.e. no spaces around braces?

Reply:
Sorry formated it by accident at some point. Will restore it back.